jobs/ci-kubernetes-e2e-gci-gce-serial-memcg.env

```diff
@@ -0,0 +1,7 @@
+### job-env
+ENABLE_GARBAGE_COLLECTOR=true
+GINKGO_TEST_ARGS=--ginkgo.focus=MemoryEviction --ginkgo.skip=
+PROJECT=k8s-jkns-ci-node-e2e
+KUBE_OS_DISTRIBUTION=gci
+
+KUBEKINS_TIMEOUT=60m
```

Comment:
then delete this env file, kubelet test envs will come from https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/jenkins/jenkins-memcg-serial.properties not here